src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,719 @@
+expense_policy_in:
+  type: object
+  required:
+    - source
+    - description
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: ./fields.yaml#/id_string
+      description: |
+        This represents the id of the expense policy.<br><br>
+        _Note: When creating the policy, this field shouldn't be set._<br>
+        _When updating, this field should be set._
+      example: 'tpr12345xyz'
+    description:
+      allOf:
+        - $ref: ./fields.yaml#/description
+      nullable: false
+      description: |
+        This represents the description of the expense policy.
+      example: Receipt is mandatory for expense above $50
+    source:
+      $ref: ./fields.yaml#/expense_policy_source
+    expense_limit_type:
+      $ref: ./fields.yaml#/expense_policy_limit_type
+    expense_currency:
+      allOf:
+        - $ref: ./fields.yaml#/currency
+      description: |
+        The three-letter currency code following ISO standard <a
+        href='https://www.iso.org/iso-4217-currency-codes.html'>ISO 4217</a>.<br>
+        _Note: When this field is set, expense_amount_max or expense_amount_min must be set._
+    expense_amount_max:
+      allOf:
+        - $ref: ./fields.yaml#/amount
+      description: |
+        Maximum amount above which policy will be violated.<br>
+        _Note: When this field is set, expense_currency must be set._
+      example: 10000
+    expense_amount_min:
+      allOf:
+        - $ref: ./fields.yaml#/amount
+      description: |
+        Minimum amount below which policy will be violated.<br>
```

Comment:
- If an expense's amount falls behind this value, this policy rule will be violated for that expense.

Reply:
applicable for all max-min cases